### DISCUSSION TITLE
LAPRAS4周年記念メッセージボード

### DISCUSSION THREAD

**Discussion by UserSuccessTeamLAPRAS** [lapras-inc/lapras-user-community#15] [General]
2023年4月10日、LAPRASはサービスのローンチから4周年を迎えました 🎉 
4年間サービスを続けて来られたのも、ひとえにユーザーの皆様のおかげです。開発チーム一同今後もサービスを進化させていきますので、ご期待ください！

### 4周年を迎えたLAPRASに是非メッセージをお願いします
ユーザーの皆様のメッセージが運営チームの力になります。激励、要望その他のコメントなど一言お待ちしています。

※ このメッセージボードは4月末まで書き込み可能です

**Comment by motuo1201** [comment]
1エンジニアとしても、採用担当としてもたくさん使わせて貰っています！
ポートフォリオが１箇所に、しかも自動でまとめてくれる発想は本当に良いですね。

これからもミスマッチのない世界を実現すべく頑張って下さい！

**Reply by UserSuccessTeamLAPRAS** [reply]
@motuo1201 
LAPRAS運営です。メッセージありがとうございます！
採用担当としてもお使いいただいているとのことで、LAPRASもLAPRAS SCOUTもこれからもっと良くなっていきますのでご期待くださいね！

**Comment by marunish** [comment]
春から使わせていただいています！

自動で取り込み評価してくれる仕組みは、初心者にも優しいです。

これからも躍進してください！

**Reply by UserSuccessTeamLAPRAS** [reply]
@marunish 
LAPRAS運営です。メッセージありがとうございます！
これからもお楽しみいただける機能を出していきますので、ご期待くださいね！

**Comment by norm02** [comment]
GitHub連携できるポートフォリオ集約サイトとして、企業とのつながりを実現するサービスとして、
愛用させていただいています！
これからもよろしくお願いします！

**Reply by UserSuccessTeamLAPRAS** [reply]
@norm02 
LAPRAS運営です。メッセージありがとうございます！
愛用していただいているとのこと、嬉しいお言葉をありがとうございます！

**Comment by KazusaNakagawa** [comment]
ポートフォリオとして愛用しております! 
今までアウトプットした内容が集約される機能が気に入ってます!!

これからのサービス進化も楽しみです🎉

**Reply by UserSuccessTeamLAPRAS** [reply]
@KazusaNakagawa 
LAPRAS運営です。メッセージありがとうございます！
愛用していただいているとのことで、ありがとうございます！ポートフォリオ機能も今後改善していきますので、よろしくお願いします！

**Comment by unsolublesugar** [comment]
4周年おめでとうござます✨🎉
いちエンジニアとして、採用担当として、公私ともにお世話になっております！

サービス開始当初から日々の機能改善、アップデートありがとうございます！
今後の発展にも期待しています〜！！😆

**Reply by UserSuccessTeamLAPRAS** [reply]
@unsolublesugar 
LAPRAS運営です。メッセージありがとうございます！
サービス開始当初からご覧いただいているようで、ありがとうございます。今後ともご期待に添えるよう、改善してまいります！

**Comment by matoruru** [comment]
4周年おめでとうございます🎉🐈
ユーザの意見を積極的に取り入れてくれる姿勢が好きです😀
日々のアウトプットが反映されていくのがとても楽しく励みになります。
さらに洗練されていくことを期待してます！

**Reply by UserSuccessTeamLAPRAS** [reply]
@matoruru 
LAPRAS運営です。メッセージありがとうございます！
LAPRASでは実際に利用いただいているユーザーの皆様の声を大切にしております。ぜひ今後もお気づきの点があればお気軽にお寄せくださいね！

**Comment by ktnyt** [comment]
こんにちは、LAPRASソフトウェアエンジニアのことねです！
内部の人間でありながらもいちLAPRASヘビーユーザーとして、ユーザー目線でのQoL改善に取り組ませていただいています。
皆様の温かいコメント大変励みになります ☺️
よりよいサービスとなるよう邁進して参りますので引き続きご意見ご要望等お聞かせ願えますと幸いです！

**Comment by ryo-endo** [comment]
こんにちは。みなさんコメントありがとうございます。
僕自身がLAPRASというサービスが大好きで、気づいたら中の人になってたので、温かいコメント嬉しいです！
今年もLAPRASは変化し続けていきますのでどうぞよろしくお願いします。
